### PR TITLE
Set winden.app version in go during build

### PR DESCRIPTION
**Pull request by donpui**
Set winden.app version in wormhole-william/version/version.go during the build.

This allows library identify as winden when connecting to mailbox server. 
We can use this later on identifying different clients using mailbox server (table: client_versions)

Additionally, I set latest git tag name in index, that it would be clear which latest git tag version we use.

### CODE REVIEW

File: client/src/app/components/Footer.tsx [hunk 1]
```diff
@@ -36,7 +36,9 @@ function Tagline() {
 
 function Logo() {
   return (
-    <Anchor href="https://leastauthority.com" target="_blank">
+    <Anchor href="https://leastauthority.com" 
+      target="_blank"
```
Comment (meejah) [inline client/src/app/components/Footer.tsx:40]: Can we please not do this? Forcing new tabs/windows is against Web recommendations too, AFAIK.

Comment (donpui) [reply client/src/app/components/Footer.tsx:40]: I am not against to change this, but can you be more specific which web recommendations? As far as I understand, this is more ethical question: give decision to users open in the same page or in new tab or decide for user, when to open in new tab/window.

Comment (meejah) [reply client/src/app/components/Footer.tsx:40]: yes, if you say target=_blank your are removing the choice and forcing a new tab/window.

https://www.w3.org/TR/WCAG20-TECHS/G200.html

Comment (donpui) [reply client/src/app/components/Footer.tsx:40]: I was trying to run some public wcag checker too, just currently didn't find one, which provides details results without registration. I will drop _blank

**Review by JustusFT — APPROVED** [review]

**Pull request merged**